src/content/core/components/namespaces.mdx

```diff
@@ -23,6 +23,21 @@ Switch your kubectl configuration to point to your namespaces with the `okteto k
 
 Once you're done, you can also delete your namespace via the UI, or with the `okteto namespace delete` command.
 
+You can sleep your namespace from the command line with `okteto namespace sleep [name]`.
+
+When you want to wake your namespace, you can use the command `okteto namespace wake [name]`.
+
+You can sleep your namespace from the Okteto UI by clicking on the Sleep dialog.
+
+In order to wake the namespace, you can click on the Wake dialog.
+
+:::tip
+
+SaaS single tenant and Self-Hosted customers can specify a global, default threshold for automatically sleeping a namespace in the [garbage collection](reference/helm-chart-values.mdx/#gc) configuration.
```

Comment:
Now that cloud is out, we only have single-tenant instances, so we can simplify the instructions: 

```suggestion
Administrators can specify a global, default threshold for automatically sleeping a namespace in the [garbage collection](reference/helm-chart-values.mdx/#gc) configuration.
```